modules/disk-uploader/pkg/certificate/certificate_test.go

```diff
@@ -0,0 +1,117 @@
+package certificate_test
+
+import (
+	"context"
+
+	. "github.com/onsi/ginkgo/v2"
+	. "github.com/onsi/gomega"
+
+	"github.com/golang/mock/gomock"
+
+	"k8s.io/apimachinery/pkg/api/errors"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+
+	v1beta1 "kubevirt.io/api/export/v1beta1"
+	kubevirtfake "kubevirt.io/client-go/generated/kubevirt/clientset/versioned/fake"
+	"kubevirt.io/client-go/kubecli"
+
+	"github.com/kubevirt/kubevirt-tekton-tasks/modules/disk-uploader/pkg/certificate"
+)
+
+var _ = Describe("Certificate", func() {
+	var (
+		vmExportClient *kubevirtfake.Clientset
+		virtClient     kubecli.KubevirtClient
+	)
+
+	const (
```

Comment:
nit: I'd put defining consts before defining vars. (applies to multiple files in this PR)